integration-tests/src/test/java/com/datastax/oss/driver/core/tracker/RequestIdGeneratorIT.java

```diff
@@ -122,4 +124,23 @@ public void should_not_write_id_to_custom_payload_when_key_is_not_set() {
       assertThat(rs.getExecutionInfo().getRequest().getCustomPayload().get("trace_key")).isNull();
```

Comment:
I think we should check here for absence of `request-id` (default), not `trace_key` (copied from upper test).